### PR TITLE
Feature merge master

### PR DESCRIPTION
**Pull request by romu4ld**
Merge master into develop.
It includes the new filter setting in motion control.
I tested app_demo_slave_ethercat_motorcontrol and it's working.

### CODE REVIEW

File: module_network_drive/src/tuning.xc [hunk 1]
```diff
@@ -313,6 +333,64 @@ void tuning_command_handler(
...
+            // set kp, ki and kd equal to 0
+            motion_ctrl_config = i_motion_control.get_motion_control_config();
+
+            motion_ctrl_config.position_kp = 0;
+            motion_ctrl_config.position_ki = 0;
+            motion_ctrl_config.position_kd = 0;
+            motion_ctrl_config.moment_of_inertia = 0;
+
+            switch(tuning_mode_state.value) {
+            case 2:
```
Comment (fjes) [inline module_network_drive/src/tuning.xc]: what does `tuning_mode_state.value == 2` mean? Maybe a define or enum can help to understand what the value `2` and `3` are for.
If there are only these two possible values maybe a if...then can be considered.

File: examples/app_demo_master_ethercat_tuning/src/tuning.c [hunk 2]
```diff
@@ -70,6 +70,15 @@ void tuning_input(struct _pdo_cia402_input pdo_input, InputValues *input)
     case TUNING_STATUS_MUX_RATED_TORQUE://rated torque
         (*input).rated_torque = pdo_input.user_miso;
         break;
+    case TUNING_STATUS_MUX_FILTER://filter
+        (*input).filter = pdo_input.user_miso;
```
Comment (fjes) [inline examples/app_demo_master_ethercat_tuning/src/tuning.c]: Here I stumbled, `(*input).filter =` is semantically the same as `input->filter`, later is more common in C.

**Comment by fjes** [comment]
@vtretyakov I was in the middle of the review ... just say'n

File: module_network_drive/src/tuning.xc (hunk 1 above)
Comment (romu4ld) [inline module_network_drive/src/tuning.xc]: yes they can be replaced by those enum values POS_PID_VELOCITY_CASCADED_CONTROLLER=2 and LT_POSITION_CONTROLLER=3.

**Pull request merged**